### DISCUSSION TITLE
Would you plan to adapt device orientation?

### DISCUSSION THREAD

**Discussion by fattiger00** [kvyatkovskys/KVKCalendar#234] [Ideas]
I notice that you set full orientation support in the example project.

But it seems not to work properly as expected
![截屏2022-03-20 17 56 36](https://user-images.githubusercontent.com/38494401/159157096-8e7ef805-4aef-44d0-aa4d-f1ea4ce96ed8.png)
![截屏2022-03-20 17 56 48](https://user-images.githubusercontent.com/38494401/159157101-07e839d8-fcb3-4510-a264-7bef94e1d4a0.png)
![截屏2022-03-20 17 57 06](https://user-images.githubusercontent.com/38494401/159157114-2b1984b1-fb94-44d7-88c5-84efd2fdad2b.png)

**Comment by kvyatkovskys** [comment]
Just call `func calendarView.reloadFrame(frame:_)` when you changing orientation

**Reply by fattiger00** [reply]
Will you consider handler this logic inside framework instead of by framework user?

**Reply by kvyatkovskys** [reply]
I don't know 🤔 I left that part to developers.

**Reply by kvyatkovskys** [reply]
Will be fixed in the swiftui version
